frontend/packages/jobs/src/functions/processSaveReview.ts

```diff
@@ -7,7 +7,7 @@ type Severity = keyof typeof SeverityEnum.enum
 
 export const processSaveReview = async (
   payload: ReviewResponse,
-): Promise<{ success: boolean }> => {
+): Promise<{ success: boolean; overallReviewId?: number }> => {
```

Comment:
There must always be a time of success!

```suggestion
): Promise<{ success: boolean; overallReviewId: number }> => {
```